runtime-sofa-boot-starter/src/main/java/com/alipay/sofa/runtime/spring/config/SofaRuntimeConfigurationProperties.java

```diff
@@ -24,19 +24,23 @@
  */
 @ConfigurationProperties(SofaRuntimeConfigurationProperties.PREFIX)
 public class SofaRuntimeConfigurationProperties {
-    static final String PREFIX                      = "com.alipay.sofa.boot";
-
-    private boolean     skipJvmReferenceHealthCheck = false;
-    private boolean     disableJvmFirst             = false;
+    static final String PREFIX = "com.alipay.sofa.boot";
 
     public void setSkipJvmReferenceHealthCheck(boolean skipJvmReferenceHealthCheck) {
-        this.skipJvmReferenceHealthCheck = skipJvmReferenceHealthCheck;
         SofaRuntimeProperties.setSkipJvmReferenceHealthCheck(this.getClass().getClassLoader(),
             skipJvmReferenceHealthCheck);
     }
 
     public void setDisableJvmFirst(boolean disableJvmFirst) {
-        this.disableJvmFirst = disableJvmFirst;
         SofaRuntimeProperties.setDisableJvmFirst(this.getClass().getClassLoader(), disableJvmFirst);
     }
+
+    public boolean isSkipJvmReferenceHealthCheck() {
+        return SofaRuntimeProperties
+            .isSkipJvmReferenceHealthCheck(this.getClass().getClassLoader());
+    }
+
+    public boolean isDisableJvmFirst() {
+        return SofaRuntimeProperties.isDisableJvmFirst(this.getClass().getClassLoader());
+    }
```

Comment:
The reason for adding getters is that the `spring-boot-configuration-processor` needs a public getter, see: https://github.com/spring-projects/spring-boot/blob/master/spring-boot-project/spring-boot-tools/spring-boot-configuration-processor/src/main/java/org/springframework/boot/configurationprocessor/ConfigurationMetadataAnnotationProcessor.java#L287 .

Reply:
@ScienJus  ok,  i would review this pr later~